### ISSUE TITLE
Due to 0.6.27, can't execute selfupate 0.6.x to 0.7.x.

### ISSUE THREAD

**Issue by hiroyuki-sato** [embulk/embulk#318]
Due to 0.6.27 release, ``embulk selfupdate`` can't work correctly.

Current implementation get latest embulk version from this URL.
```
https://bintray.com/embulk/maven/embulk/_latestVersion
```

After request, It get version number from Location header. 

Now latest is *0.6.27*. 

```
curl -v https://bintray.com/embulk/maven/embulk/_latestVersion
*   Trying 119.81.184.202...
* Connected to bintray.com (119.81.184.202) port 443 (#0)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
* Server certificate: *.bintray.com
* Server certificate: GeoTrust SSL CA - G4
* Server certificate: GeoTrust Global CA
> GET /embulk/maven/embulk/_latestVersion HTTP/1.1
> Host: bintray.com
> User-Agent: curl/7.43.0
> Accept: */*
> 
< HTTP/1.1 302 Found
< Server: nginx
< Date: Thu, 01 Oct 2015 07:11:54 GMT
< Content-Length: 0
< Connection: keep-alive
< Location: https://bintray.com/embulk/maven/embulk/0.6.27/view
< 
* Connection #0 to host bintray.com left intact
```

Need fix this code, or release 0.7.5.

https://github.com/embulk/embulk/blob/master/lib/embulk/command/embulk_selfupdate.rb#L85

**Comment by frsyuki** [comment]
Thank you for reporting. I released v0.7.5.

**Comment by hiroyuki-sato** [comment]
Thanks!
